set.mm

```diff
@@ -73922,6 +73936,30 @@ Relations and functions (cont.)
       ( cdm cvv fndmd dmexd eqeltrrd ) ACGBHABCFIACDEJK $.
   $}
 
+  $( The domain of a function is a set iff the function is a set.  (Contributed
+     by AV, 8-Aug-2024.) $)
+  fndmexb $p |- ( F Fn A -> ( A e. _V <-> F e. _V ) ) $=
+    ( wfn cvv wcel fnex ex wa simpr simpl fndmexd impbid ) BACZADEZBDEZMNOADBFG
+    MONMOHABDMOIMOJKGL $.
+
+  $( The domain of a function is a set iff the function is a set.  (Contributed
+     by AV, 8-Aug-2024.) $)
+  fdmexb $p |- ( F : A --> B -> ( A e. _V <-> F e. _V ) ) $=
+    ( wf wfn cvv wcel wb ffn fndmexb syl ) ABCDCAEAFGCFGHABCIACJK $.
+
+  $( If a mapping is a set, its domain is a set.  (Contributed by NM,
+     27-Aug-2006.)  (Proof shortened by Andrew Salmon, 17-Sep-2011.) $)
+  dmfex $p |- ( ( F e. C /\ F : A --> B ) -> A e. _V ) $=
+    ( wf wcel cvv cdm wceq wi fdm dmexg eleq1 syl5ib syl impcom ) ABDEZDCFZAGFZ
+    QDHZAIZRSJABDKRTGFUASDCLTAGMNOP $.
```

Comment:
It seems more natural to swap the order of these two theorems: first dmfex, and then fdmexb.

Reply:
Done.